nipype/interfaces/afni/preprocess.py

```diff
@@ -2789,6 +2790,13 @@ class WarpInputSpec(AFNICommandInputSpec):
         argstr='-zpad %d')
     verbose = traits.Bool(
         desc='Print out some information along the way.', argstr='-verb')
+    save_warp = traits.Bool(
+        False, usedefault=True, desc='save warp as .mat file')
```

Comment:
The default of a `Bool` is already false. You can just have `desc` here.

Reply:
Oh, and RE your initial comment, I would set `requires=['verbose']`. I'm not sure what will happen if someone sets `verbose=False`, but at least it will catch when `verbose` is unset.